test-poc/framework/src/main/java/org/keycloak/test/framework/server/KeycloakTestServerSupplier.java

```diff
@@ -22,11 +24,12 @@ public Class<KeycloakIntegrationTest> getAnnotationClass() {
     @Override
     public InstanceWrapper<KeycloakTestServer, KeycloakIntegrationTest> getValue(Registry registry, KeycloakIntegrationTest annotation) {
         KeycloakTestServerConfig serverConfig = SupplierHelpers.getInstance(annotation.config());
+        DatabaseConfig databaseConfig = annotation.dbconfig() != null ? SupplierHelpers.getInstance(annotation.dbconfig()) : new DefaultDatabaseConfig();
```

Comment:
I think it can be left like this because the `dbconfig()` returns the default value (`DefaultDatabaseConfig.class`) if it is not entered. 

```suggestion
        DatabaseConfig databaseConfig = SupplierHelpers.getInstance(annotation.dbconfig());
```